src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,67 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../types/Post';
+import { User } from '../../types/User';
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  posts: Post[],
+  comments: Comment[],
+  users: User[],
+};
+
+function createPostsInfo(
+  posts: Post[],
+  comments:Comment[],
+  users: User[],
+) {
+  const postsInfo = posts.map(post => {
+    const user = users.find(
```

Comment:
Create a separate functions to find users and posts to make this `createPostsInfo` function simpler